e2b-dev/e2b
# CLI basic commands

Testing

* Run API on port 3003 (or set E2B_API_BASE env var when running CLI commands)
* build CLI
* cd into `packages/cli/testground/demo-basic`
* run ` ./../../dist/index.js env create`

Expected output:
![Screen 2023-09-14 at 13 05 49](https://github.com/e2b-dev/e2b/assets/697301/d63c9e60-9485-4d3f-9ae0-ef5e9980c33c)
